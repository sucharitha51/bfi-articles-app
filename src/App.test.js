import React from 'react';
import { shallow } from 'enzyme'  
import App from './App';

let wrapper

beforeEach(() => {
  wrapper = shallow(<App />)
})

describe('App  component', () => {

  it('App renders without crashing', () => {
    expect(wrapper.exists()).toEqual(true);
  });

  it('App should match with snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
})