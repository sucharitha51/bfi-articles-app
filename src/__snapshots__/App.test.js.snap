// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  component App should match with snapshot 1`] = `"<div><div class=\\"nav-bar\\"><ul><li><a href=\\"/\\"><b>Home</b></a></li><li><a href=\\"/review\\"><b>Reviews</b></a></li><li><a href=\\"/features\\"><b>Features</b></a></li><li><a href=\\"/video\\"><b>Videos</b></a></li></ul></div><h2><a href=\\"/latest\\">Latest</a></h2><div class=\\"archive\\"></div><h2><a href=\\"/reviews\\">Reviews</a></h2><div class=\\"archive\\"></div><h2><a href=\\"/videos\\">Videos</a></h2><div class=\\"archive\\"></div><h2><a href=\\"/features\\">Features</a></h2><div class=\\"archive\\"></div></div>"`;
